roles/manage-aws-infra/templates/casl-aws-cformation.yml.j2

```diff
@@ -676,6 +734,9 @@ Resources:
     {% for idx in range(0, aws_num_etcd | int) -%}
     - ETCD{{ '%02d' % idx }}
     {% endfor -%}
+    {% for idx in range(0, aws_num_bastion | int) -%}
+    - BAST{{ '%02d' % idx }}
```

Comment:
Check that you are declaring the bastion objects as 'Bastion{{ '%02d' % idx }}', not 'BAST{{ '%02d' % idx }}'. Please change this.